player.py

```diff
@@ -3,41 +3,63 @@
 #player tank
 class Player(pygame.sprite.Sprite):
     def __init__(self, x, y, width, height):
-        #location
-        self.x = x
-        self.y = y
+        super().__init__()
+
+        # image = pygame.image.load(TANK_UP)
+        self.image = pygame.transform.scale(pygame.image.load(TANK_UP), (width, height))
+        self.direction = TANK_UP
+
         #size
         self.width = width
         self.height = height
+
+        #starting coordinates for the player 
+            #(not the player image)
+        self.x = x
+        self.y = y
```

Comment:
It doesn't seem like you use these variables anywhere else but to assign to self.rect.  Why not just skip this step and assign x and y directly to self.rect?

Reply:
yeah I thought something else was relying on this but it doesn't seem like it is. This should have been refactored after the graphic was created. good catch